.github/ISSUE_TEMPLATE/introduce-sgid-dataset.md

```diff
@@ -83,7 +83,7 @@ assign yourself or someone to check that the dataset is live in its area. once v
 - [ ] Stewardship doc and sgid-index (@assigned on ``)
 - [ ] `SGID.META.AGOLItems` (@assigned on ``)
 - [ ] `SGID.META.ChangeDetection` (@assigned on ``)
-- [ ] `agol-validator` items (tags, thumbnail, etc) (@assgined on ``)
+- [ ] [agol-validator](https://github.com/agrc/agol-validator) items (@assgined on ``)
```

Comment:
**question**: this seems be better organized in the "data is high quality" section since the agol validator isn't a place that the data lives. But the "data is high quality" are pre publish steps. Should we introduce a post publish section or is that too pedantic?

Reply:
Or maybe we change the term from `agol-validator` to `Standardized ArcGIS Online` or something like that?